.k8s_ci.Dockerfile

```diff
@@ -41,21 +41,15 @@ ENV LANG=en_US.UTF-8
 
 # Install build dependencies + ansible
 RUN set -xe; \
-    echo "force-unsafe-io" > /etc/dpkg/dpkg.cfg.d/02apt-speedup \
-    && echo "Acquire::http {No-Cache=True;};" > /etc/apt/apt.conf.d/no-cache \
+    echo "Acquire::http {No-Cache=True;};" > /etc/apt/apt.conf.d/no-cache \
```

Comment:
My understanding was that using unsafe io would only be a problem in the event of a power loss, which is a non-issue when building a docker image? This was copied over from the docker-galaxy-stable image and It does provide a significant reduction in build time.

Reply:
I see. Maybe I overreacted on the word "unsafe".

Reply:
I benchmarked it with `time` and found no difference on my machine.